beets/autotag/hooks.py

```diff
@@ -332,7 +332,7 @@ def __init__(self):
         self._penalties = {}
 
     @LazyClassProperty
-    def _weights(cls):  # noqa
+    def _weights(self, cls):  # noqa
```

Comment:
I don't think this is correct—the decorator only provides the class (like `@classmethod` or similar).

Reply:
Not completely sure, flake8 made the suggestion.